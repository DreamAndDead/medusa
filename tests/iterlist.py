a = [1, 4, 8, 16, 52, 34, 78, 342]

k = 0

strange_sum = 0
for i in a:
    k += 1
    if k > 3:
        break

    print("Current i is: ", i)
    strange_sum += i

print("After break: ")

for i in a:
    print("Current i is: ", i)
    strange_sum += i

print("Some strange sum is: ", strange_sum)