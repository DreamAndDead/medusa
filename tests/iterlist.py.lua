dofile('./pythonlua/luainit.lua')

local a = list {1, 4, 8, 16, 52, 34, 78, 342}
local k = 0
local strange_sum = 0
for i in a do
    k = (k + 1)
    if (k > 3) then
        break
    end
    print("Current i is: ", i)
    strange_sum = (strange_sum + i)
    ::loop_label_1::
end
print("After break: ")
for i in a do
    print("Current i is: ", i)
    strange_sum = (strange_sum + i)
    ::loop_label_2::
end
print("Some strange sum is: ", strange_sum)