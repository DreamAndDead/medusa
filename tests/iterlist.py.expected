Current i is: 	1
Current i is: 	4
Current i is: 	8
After break: 
Current i is: 	1
Current i is: 	4
Current i is: 	8
Current i is: 	16
Current i is: 	52
Current i is: 	34
Current i is: 	78
Current i is: 	342
Some strange sum is: 	548
